sdk/communication/Azure.Communication.PhoneNumbers/README.md

```diff
@@ -30,21 +30,31 @@ Here's an example using the Azure CLI:
 
 ### Key concepts
 
-Phone plans come in two types; Geographic and Toll-Free. Geographic phone plans are phone plans associated with a location, whose phone numbers' area codes are associated with the area code of a geographic location. Toll-Free phone plans are phone plans not associated location. For example, in the US, toll-free numbers can come with area codes such as 800 or 888.
+This SDK provides functionality to easily manage `direct offer` and `direct routing` numbers.
 
-All geographic phone plans within the same country are grouped into a phone plan group with a Geographic phone number type. All Toll-Free phone plans within the same country are grouped into a phone plan group.
+The `direct offer` numbers come in two types: Geographic and Toll-Free. Geographic phone plans are phone plans associated with a location, whose phone numbers' area codes are associated with the area code of a geographic location. Toll-Free phone plans are phone plans not associated location. For example, in the US, toll-free numbers can come with area codes such as 800 or 888.
+They are managed using the `PhoneNumbersClient`
+
+The `direct routing` feature enables connecting your existing telephony infrastructure to ACS.
+The configuration is managed using the `SipRoutingClient`, which provides methods for setting up SIP trunks and voice routing rules, in order to properly handle calls for your telephony subnet.
 
 ### Authenticate the client
 
-Phone Number clients can be authenticated using connection string acquired from an Azure Communication Resources in the [Azure Portal][azure_portal].
+Clients can be authenticated using connection string acquired from an Azure Communication Resources in the [Azure Portal][azure_portal].
 
 ```C# Snippet:CreatePhoneNumbersClient
 // Get a connection string to our Azure Communication resource.
 var connectionString = "<connection_string>";
 var client = new PhoneNumbersClient(connectionString);
 ```
 
-Phone Number clients also have the option to authenticate with Azure Active Directory Authentication. With this option,
+```C# Snippet:CreateSipRoutingClient
+// Get a connection string to Azure Communication resource.
+var connectionString = "<connection_string>";
+var client = new SipRoutingClient(connectionString);
+```
+
+Clients also have the option to authenticate with Azure Active Directory Authentication. With this option,
```

Comment:
Do these variables need to be set, or is this only if they explicitly want to use the `EnvironmnetCredential`? Perhaps we should just show the `DefaultAzureCredential` sample and link to the Azure.Identity readme for more credential options.